controller/src/command.cpp

```diff
@@ -141,15 +141,29 @@ void command_execute(enum command cmd, char *dataTx, uint8_t lenTx,
   case command::comms_check:
     break;
   case command::set_ventilatorMode:
-    parameters_setVentilatorMode((enum ventilatorMode)dataTx[0]);
+    /* Only allow one to change the ventilator mode when it isn't running */
+    if(respirationCtrl_getVentilatorState() == ventilatorState::not_running) {
+        parameters_setVentilatorMode((enum ventilatorMode) dataTx[0]);
+    }
```

Comment:
This will probably require a way to alert the user that his command was not properly executed...